From 22200891efea69eb17772da8ae5e074860ba1ad4 Mon Sep 17 00:00:00 2001
From: George Matsumura <gmmatsumura01@bvsd.org>
Date: Sun, 6 Oct 2019 17:43:24 +0000
Subject: [PATCH xf86-video-mach64 3/4] Fix debugging traces

Without this change, the compiler emits an error about pPict being undefined when fallback messages are turned on.
---
 src/atimach64render.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git src/atimach64render.c src/atimach64render.c
index 8d259fa..e4ddea8 100644
--- a/src/atimach64render.c
+++ b/src/atimach64render.c
@@ -701,8 +701,8 @@ Mach64PrepareComposite
             break;
     }
     if (i == MACH64_NR_TEX_FORMATS)
-        MACH64_FALLBACK(("Unsupported picture format 0x%x\n",
-                        (int)pPict->format));
+        MACH64_FALLBACK(("Unsupported dst format 0x%x\n",
+                        (int)pDstPicture->format));
     dstFormat = Mach64TexFormats[i].dstFormat;
 
     m3d->dp_pix_width = SetBits(dstFormat, DP_DST_PIX_WIDTH) |
-- 
2.25.0

